src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetProcessName.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
```

Comment:
Do we really need that? I would expect `nullable` to be enabled by default in the entire repo.

```suggestion
```